apps/transport/lib/jobs/gtfs_diff_job.ex

```diff
@@ -37,7 +39,9 @@ defmodule Transport.Jobs.GTFSDiff do
 
       Oban.Notifier.notify(Oban, :gossip, %{
         complete: job.id,
-        diff_file_url: Transport.S3.permanent_url(:gtfs_diff, diff_file_name)
+        diff_file_url: Transport.S3.permanent_url(:gtfs_diff, diff_file_name),
+        gtfs_file_name_1: gtfs_original_file_name_1,
+        gtfs_file_name_2: gtfs_original_file_name_2
```

Comment:
```suggestion
        gtfs_original_file_name_1: gtfs_original_file_name_1,
        gtfs_original_file_name_2: gtfs_original_file_name_2
```

Pour garder le même nom de variable tout le long et éviter une confusion.

Reply:
En effet.